[2025-09-10T13:10:45.127+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-10T13:10:45.236+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.load manual__2025-09-10T13:10:33.763732+00:00 [queued]>
[2025-09-10T13:10:45.257+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.load manual__2025-09-10T13:10:33.763732+00:00 [queued]>
[2025-09-10T13:10:45.258+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-09-10T13:10:45.311+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): load> on 2025-09-10 13:10:33.763732+00:00
[2025-09-10T13:10:45.335+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'load', 'manual__2025-09-10T13:10:33.763732+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmpt8ziurg0']
[2025-09-10T13:10:45.341+0000] {standard_task_runner.py:105} INFO - Job 156: Subtask load
[2025-09-10T13:10:45.341+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2418) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-10T13:10:45.343+0000] {standard_task_runner.py:72} INFO - Started process 2419 to run task
[2025-09-10T13:10:45.654+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.load manual__2025-09-10T13:10:33.763732+00:00 [running]> on host b37bbfb35b9c
[2025-09-10T13:10:46.112+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-09-10T13:10:33.763732+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-10T13:10:33.763732+00:00'
[2025-09-10T13:10:46.114+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-10T13:10:46.146+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-10T13:10:46.149+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/load.py']
[2025-09-10T13:10:46.180+0000] {subprocess.py:86} INFO - Output:
[2025-09-10T13:10:49.508+0000] {subprocess.py:93} INFO - Loaded to DB
[2025-09-10T13:10:49.713+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-10T13:10:49.961+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-10T13:10:49.963+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=load, run_id=manual__2025-09-10T13:10:33.763732+00:00, execution_date=20250910T131033, start_date=20250910T131045, end_date=20250910T131049
[2025-09-10T13:10:50.042+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-10T13:10:50.172+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-10T13:10:50.197+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-09-10T13:10:50.350+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
